starter_scripts/exaslct_within_docker_container.sh

```diff
@@ -93,6 +94,7 @@ tmpfile_env=$(mktemp)
 trap 'rm -f -- "$tmpfile_env"' INT TERM HUP EXIT
 
 create_env_file_debug_protected "$tmpfile_env"
+# shellcheck disable=SC2068
```

Comment:
I thought the double quotes around array expansion actually would work, but not sure about it

Reply:
done